docker-compose.yaml

```diff
@@ -21,6 +21,7 @@ services:
       - ./grafana/etc:/etc/grafana:ro
       - grafana-lib:/var/lib/grafana
       - grafana-log:/var/log/grafana
+      - ./grafana/etc/grafana.ini:/etc/grafana/grafana.ini
```

Comment:
Originally, I put `:ro` to ensure all changes are version controlled.
To make it possible to edit afterward, simply remove the ending `:ro` annotation at Line 21.

Is that what you are trying to do?

Reference for the syntax v3:
- https://docs.docker.com/compose/compose-file/compose-file-v3/#short-syntax-3

Reply:
Certainly, that method is effective. May I kindly suggest adding a comment to the code to make it more accessible for individuals who may not be familiar with the concept? This will greatly benefit others who may come across the code in the future.

Reply:
Will create another PR for adding comments :)